NAMESPACE

```diff
@@ -4,6 +4,7 @@ S3method(format,orderly_query)
 S3method(print,orderly_cleanup_status)
 S3method(print,orderly_query_explain)
 export(orderly_artefact)
+export(orderly_build_script_details)
```

Comment:
What about calling it `orderly_parse()` and have it take arguments of _either_ `path` or `expr`, then just documenting it as experts use? I can see this being generally useful really

Reply:
Yep I like that